Sources/GRPCHTTP2Core/Server/HTTP2ServerTransport.swift

```diff
@@ -188,4 +188,49 @@ extension HTTP2ServerTransport.Config {
       Self(maxRequestPayloadSize: 4 * 1024 * 1024)
     }
   }
+
+  public struct TLS: Sendable {
+    public struct Certificate: Sendable {
```

Comment:
This type has no public `init`

Reply:
I'm also not sure we should name it `Certificate`, it's really a serialized certificate. (e.g. compare with https://github.com/apple/swift-certificates/blob/main/Sources/X509/Certificate.swift)